src/infohudmodes.asm

```diff
@@ -735,8 +736,16 @@ status_vspeed:
   .newjump
     LDA #$0001 : STA !ram_roomstrat_counter : STA !ram_walljump_counter
 
+    ; Print initial jump speed over item%
```

Comment:
There's the max etanks number below item%.  Perhaps we could Draw3 the whole number speed (upper byte of 0B2D) over item% and then Draw2Hex the fractional speed (lower byte of 0B2D) over the max etanks.

Reply:
The whole number would always be a single digit afaik. I haven't seen it go over 9 yet. I'd do it if you think it would be easier for users to figure out.

Reply:
Yeah that's the main reason I was suggesting it, just easier to understand

...although it might be more obnoxious to read... maybe it's fine the way you have it.